en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-300-to-310.md

```diff
@@ -0,0 +1,713 @@
+    
+    ```
+    [registry.static_configuration]
+    enable=true
+    ```
+    
+    !!! note "NOTE"
+        Changing these configuration should only be done before the initial API-M Server startup. If changes are done after the initial startup, the registry resource created previously will not be available.
+
+!!! note
+    If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.10.0](../upgrading-wso2-is-as-key-manager/upgrading-from-is-km-590-to-5100.md).
+
+-   [Step 1 - Migrate the API Manager configurations](#step-1-migrate-the-api-manager-configurations)
+-   [Step 2 - Upgrade API Manager to 3.1.0](#step-2-upgrade-api-manager-to-300)
+-   [Step 3 - Optionally, migrate the configurations for WSO2 API-M Analytics](#step-3-optionally-migrate-the-configurations-for-wso2-api-m-analytics)
+-   [Step 4 - Restart the WSO2 API-M 3.1.0 server](#step-4-restart-the-wso2-api-m-300-server)
+
+### Step 1 - Migrate the API Manager configurations
+
+!!! warning
+    Do not copy entire configuration files from the current version of WSO2 API Manager to the new one, as the configuration modal has been changed and now all the configurations are being done via a single file (deployment.toml). Instead, redo the configuration changes in the new configuration file. For more information refer [Configuration Catalog](../../reference/ConfigCatalog.md).
+
+Follow the instructions below to move all the existing API Manager configurations from the current environment to the new one.
+
+1.  Back up all databases in your API Manager instances along with the Synapse configurations of all the tenants and the super tenant.
+
+    -   The Synapse configurations of the super tenant are in the `<OLD_API-M_HOME>/repository/deployment/server/synapse-configs/default` directory.
+
+    -   The Synapse configurations of tenants are in the `<OLD_API-M_HOME>/repository/tenants` directory.
+
+    -   If you use a **clustered/distributed API Manager setup** , back up the available configurations in the **API Gateway** node.
+
+2.  Download [WUM updated](https://docs.wso2.com/display/updates/Getting+Started) pack for [WSO2 API Manager 3.1.0](http://wso2.com/api-management/).
+
+3.  Open the `<API-M_3.1.0_HOME>/repository/conf/deployment.toml` file and provide the datasource configurations for the following databases.
+
+    -   User Store
+    -   Registry database/s
+    -   API Manager databases
+
+    !!! note
+        In API-M 3.1.0, a combined SHARED_DB has been introduced to keep both the user related data (`WSO2UM_DB`) and the registry data (`WSO2REG_DB`). If you have used separate DBs for user management and registry in the previous version, you need to configure WSO2REG_DB and WSO2UM_DB databases separately in API-M 3.1.0 to avoid any issues.
```

Comment:
Shall we remove the line saying - `In API-M 3.1.0, a combined SHARED_DB has been introduced to keep both the user related data (WSO2UM_DB) and the registry data (WSO2REG_DB). ` as it is the same behavior in 3.0.0

Reply:
Yes removed it. But kept the remaining part as there might be users who still uses two separate DBs. Resolved with 02d0d1b869f0f90576477aead17e8dfa5c919384